packages/react-core/src/components/Select/Select.tsx

```diff
@@ -1250,7 +1250,7 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
         ref={this.parentRef}
         {...getOUIAProps(Select.displayName, ouiaId !== undefined ? ouiaId : this.state.ouiaStateId, ouiaSafe)}
         {...(width && { style: { width } })}
-        {...(validated !== ValidatedOptions.default && { 'aria-describedby': ariaDescribedby })}
+        {...(ariaDescribedby !== '' && { 'aria-describedby': ariaDescribedby })}
```

Comment:
```suggestion
        {...(ariaDescribedby && { 'aria-describedby': ariaDescribedby })}
```

Reply:
Thanks for the comments. 

If we relocate `ariaDescribedby` to `<input>`s, we will be unable to use it with different variants of `Select` as some of them do not have inputs. 

I see 2 possibilities how to avoid this:

1. Add `aria-describedby` to `<input>`s and keep it also where it is now
2. Add new prop `typeAheadAriaDescribedBy`, apply it to `<input>`s and keep `aria-describedby` where it is now. Similarly to [typeAheadAriaLabel](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Select/Select.tsx#L1321)

I would prefer the second option, what about you @jenny-s51?

Reply:
Thank you for clarifying @tompsota - I like the second option as well! It would make sense to follow this existing convention the way it's used for typeAheadAriaLabel